charts/mageai/templates/configmap.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
Does it throw error when .Values.config is not defined? I want to make sure it's backward compatible.

Reply:
No it doesnt, as it will also create a configmap and secret if there is no data. 
But I get the point. Ill change it so that it will
- create the configmap and secret
- set the envFrom

 only if there is data (envs) specified